PyU4V/replication.py

```diff
@@ -1,661 +1,675 @@
-            options=suspend_options, _async=_async)
-
-    def establish_storagegroup_srdf(
-            self, storagegroup_id, rdfg_no,
-            establish_options=None, _async=False):
-        """Establish io on the links for the given storagegroup.
-
-        Optional parameters to set are "bypass", "metroBias", "star",
-        "hop2", "force", "symForce", "full" - all true/false.
-
-        :param storagegroup_id: the storagegroup id
-        :param rdfg_no: the rdf group no
-        :param establish_options: Optional dict of establish params
-        :param _async: flag to indicate async call
-        """
-        return self.modify_storagegroup_srdf(
-            storagegroup_id, 'establish', rdfg_no,
-            options=establish_options, _async=_async)
-
-    def failover_storagegroup_srdf(
-            self, storagegroup_id, rdfg_no,
-            failover_options=None, _async=False):
-        """Failover a given storagegroup.
-
-        Optional parameters to set are "bypass", "star", "restore",
-        "immediate", "hop2", "force", "symForce", "remote", "establish" - all
-        true/false.
-
-        :param storagegroup_id: the storagegroup id
-        :param rdfg_no: the rdf group no
-        :param failover_options: Optional dict of failover params
-        :param _async: flag to indicate async call
-        """
-        return self.modify_storagegroup_srdf(
-            storagegroup_id, 'failover', rdfg_no,
-            options=failover_options, _async=_async)
-
-    def failback_storagegroup_srdf(
-            self, storagegroup_id, rdfg_no, failback_options=None,
-            _async=False):
-        """Failback a given storagegroup.
-
-        Optional parameters to set are "bypass", "recoverPoint", "star",
-        "hop2", "force", "symForce", "remote" - all true/false.
-
-        :param storagegroup_id: the storagegroup id
-        :param rdfg_no: the rdf group no
-        :param failback_options: Optional dict of failover params
-        :param _async: flag to indicate async call
-        """
-        return self.modify_storagegroup_srdf(
-            storagegroup_id, 'failback', rdfg_no,
-            options=failback_options, _async=_async)
-
-    def delete_storagegroup_srdf(
-            self, storagegroup_id, rdfg_num=None):
-        """Delete srdf pairings for a given storagegroup.
-
-        :param storagegroup_id:
-        :param rdfg_num: the rdfg number to remove pairings from - can be list
-        """
-        # Get a list of SRDF groups for storage group
-        if not rdfg_num:
-            rdfg_num = self.get_storagegroup_srdfg_list(storagegroup_id)
-        if not isinstance(rdfg_num, list):
-            rdfg_num = [rdfg_num]
-        for rdfg in rdfg_num:
-            res_name = "%s/rdf_group/%s" % (storagegroup_id, rdfg)
-            self.delete_resource(
-                self.array_id, REPLICATION,
-                'storagegroup', resource_name=res_name)
+# The MIT License (MIT)

```

Comment:
Will review once line endings are fixed.

Reply:
File has been converted to unix